pkg/controller/storageversiongc/gc_controller_test.go

```diff
@@ -48,7 +48,7 @@ func newKubeApiserverLease(name, holderIdentity string) *coordinationv1.Lease {
 			Name:      name,
 			Namespace: metav1.NamespaceSystem,
 			Labels: map[string]string{
-				"k8s.io/component": "kube-apiserver",
+				"apiserver.kubernetes.io/identity": "kube-apiserver",
```

Comment:
We use `internal.apiserver.k8s.io` as the group of the storage version API.  Should we use that here?

Reply:
From @sftim's comment in https://github.com/kubernetes/kubernetes/issues/114314, I got the impression that we use `*.kubernetes.io` for labels and annotations, hence the concerns around the current label `k8s.io/component`

Reply:
See https://kubernetes.io/docs/reference/labels-annotations-taints/ for a list of what's already registered.

Reply:
I'd use `kubernetes.io/api-server-identity`. That key includes everything you need to know about the label / labelled Lease, and we don't have to worry about being an out-of-tree implementation that needs to avoid the risk of a collision with something in-tree.

If we want to, we can also _annotate_ the lease with more details such as the unhashed hostname and the name that the component would like to assert (eg: `kube-apiserver`).

`apiserver.kubernetes.io/identity` is also valid.

Reply:
For code things, we tend to write `apiserver` not `api-server`. I don't really have a strong opinion about the exact label.